staging/src/k8s.io/apiserver/pkg/server/patch_genericapiserver.go

```diff
@@ -116,7 +116,9 @@ func WithNonReadyRequestLogging(handler http.Handler, hasBeenReadyCh <-chan stru
 		// ignore connections to local IP. Those clients better know what they are doing.
 		if pth := "/" + strings.TrimLeft(r.URL.Path, "/"); pth != "/readyz" && pth != "/healthz" && pth != "/livez" {
 			if isLocal(r) {
-				klog.V(2).Infof("Loopback request to %q (user agent %q) before server is ready. This client probably does not watch /readyz and might get inconsistent answers.", r.URL.Path, r.UserAgent())
+				if !isKubeApiserverLoopBack(r) {
```

Comment:
you could change it to if `isLocal(r) && !isKubeApiserverLoopBack(r)`

Reply:
no, then the else case is hit and it shouldn't.

Reply:
ah, ic, correct.